### PR TITLE
refactoring the code

### PR DESCRIPTION
**Pull request by brunobgf**

### CODE REVIEW

**Comment by brunobgf** [comment]
Acabei deletando uma pasta que não devia no segundo commit, então tive que arrumar tudo nos commits seguintes :(

File: todo/lib/src/pages/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  var items = <Item>[];
+
+  @override
+  void initState() {
+    super.initState();
+    loadItems();
+  }
+
+  void addItem() {
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: make the method private (_addItem)

File: todo/lib/src/pages/home_page.dart [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        title: newTaskController.text,
+        done: false,
+      ));
+
+      newTaskController.text = "";
+      saveItems();
+    });
+  }
+
+  void removeItem(int index) {
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: make the method private (_removeItem)

File: todo/lib/src/pages/home_page.dart [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    setState(() {
+      items.removeAt(index);
+      saveItems();
+    });
+  }
+
+  Future loadItems() async {
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: make the method private (_loadItems)

File: todo/lib/src/pages/home_page.dart [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    if (data != null) {
+      Iterable decoded = jsonDecode(data);
+      List<Item> result = decoded.map((task) => Item.fromJson(task)).toList();
+      setState(() {
+        items = result;
+      });
+    }
+  }
+
+  saveItems() async {
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: make the method private (_saveItems)

File: todo/lib/src/pages/home_page.dart [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        itemBuilder: (BuildContext context, int index) {
+          final item = items[index];
+          return Dismissible(
+            key: Key("${item.title}"),
+            background: Container(
+              color: Colors.red.withOpacity(0.2),
+            ),
+            onDismissed: (direction) => removeItem(index),
+            child: CheckboxListTile(
+              title: Text("${item.title}"),
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: add comma after string

File: todo/lib/src/pages/home_page.dart [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+                  saveItems();
+                });
+              },
+            ),
+          );
+        },
+      ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: addItem,
+        child: Icon(Icons.add),
```
Comment (labtxcelo) [inline todo/lib/src/pages/home_page.dart]: add comma after "Icons.add"

Comment (brunobgf) [reply todo/lib/src/pages/home_page.dart]: Thanks for all of the reviews!! o/

**Pull request merged**